src/cascadia/TerminalSettingsModel/TerminalSettingsSerializationHelpers.h

```diff
@@ -434,11 +434,14 @@ JSON_ENUM_MAPPER(::winrt::Microsoft::Terminal::Settings::Model::SplitType)
 
 JSON_ENUM_MAPPER(::winrt::Microsoft::Terminal::Settings::Model::SettingsTarget)
 {
-    JSON_MAPPINGS(4) = {
+    JSON_MAPPINGS(7) = {
         pair_type{ "settingsFile", ValueType::SettingsFile },
         pair_type{ "defaultsFile", ValueType::DefaultsFile },
         pair_type{ "allFiles", ValueType::AllFiles },
         pair_type{ "settingsUI", ValueType::SettingsUI },
+        pair_type{ "fileExplorer", ValueType::FileExplorer },
+        pair_type{ "sendInput", ValueType::SendInput },
+        pair_type{ "clipboard", ValueType::Clipboard },

```

Comment:
Discussed this as a team. I see value in opening the directory, but `sendInput` and `clipboard` might be too niche. We'd prefer that you back out those two values please.